packages/flutter/lib/src/widgets/banner.dart

```diff
@@ -301,6 +313,9 @@ class Banner extends StatelessWidget {
   ///  * [textDirection], which controls the reading direction of the [message].
   final TextDirection layoutDirection;
 
+  /// Height of banner.
+  final double height;
```

Comment:
It might be better to rename this to `thickness` as to not confuse it with the total dimensions of what is painted, what do you think?